.cz-config.js

```diff
@@ -4,7 +4,7 @@ const getPackages = () => {
   let outputPackages;
 
   try {
-    outputPackages = execSync('npx turbo run build --dry=json');
+    outputPackages = execSync('npx turbo run build --dry=json --ignore=nitro-output');
```

Comment:
what is `nitro-output?

Reply:
Perhaps it's worth adding a comment for the reason for using that flag.